app/models/billable_metric.rb

```diff
@@ -67,6 +67,17 @@ def self.ransackable_attributes(_auth_object = nil)
     %w[name code]
   end
 
+  # by default rails would return " Validation failed: Field name value_is_mandatory", which is confusing
+  # as model also has "name" attribute. This method overrides the default behavior to return "field_name" instead.
+  def self.human_attribute_name(attr, options = {})
+    case attr.to_s
+    when "field_name"
+      "field_name"
```

Comment:
this actually breaks the expected behaviour of the humanized attribute name... for example, "first_name" will be rendered as "First name" while this one will be "field_name".

For the "name" attribute it should read `Validation failed: Name value_is_mandatory`. See, the word "field" is not part of rails.

I would keep the default behaviour.